test/typescript/enhancers.ts

```diff
@@ -69,7 +78,13 @@ function stateExtension() {
           extraField: 'extra'
         }
       : undefined
-    return createStore(wrappedReducer, wrappedPreloadedState)
+    const store = createStore(wrapReducer(reducer), wrappedPreloadedState)
+    return {
+      ...store,
+      replaceReducer(nextReducer: Reducer<S, A>) {
+        store.replaceReducer(wrapReducer(nextReducer))
+      }
+    }
```

Comment:
The change in types actually caught a bug in one the tests where we weren't re-wrapping the reducer previously.